### DISCUSSION TITLE
AQI from Davis Airlink

### DISCUSSION THREAD

**Discussion by OE3MNS** [poblabs/weewx-belchertown#950] [Q&A]
Hello everyone, I would like to use the AQI data displayed in Skin Belchertown not from the API but from Davis Airlink (pm2_5_api) and have the update carried out via MQTT. Unfortunately, my knowledge of writing code is not enough for that. Perhaps someone could please tell me what I need to rewrite in belchertown.js.tmpl.

Many thanks

**Comment by c0r3dump3d** [comment]
Hi, you can use this weewx plugin:

https://github.com/chaunceygardiner/weewx-airlink

And then configure some graphics in Belchertown skin like this:

![imagen](https://github.com/user-attachments/assets/8064f2e4-c565-4bf4-af0c-d6e07c56af08)

![imagen](https://github.com/user-attachments/assets/6013149b-056b-48ca-a139-30029b30cfb2)
